src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/helpers.js

```diff
@@ -0,0 +1,99 @@
+import moment from 'moment';
```

Comment:
Just experimenting with a separate helper file for each test. I could see this method just living in the test file if we don't want to create a bunch of these files.